.github/workflows/create-deployment-spack.yml

```diff
@@ -0,0 +1,46 @@
+name: Create Deployment Spack
+on:
+  workflow_dispatch:
+    inputs:
+      spack-version:
+        type: string
+        required: true
+        description: A version of spack
+      spack-packages-version:
+        type: string
+        required: true
+        default: main
+        description: A version of ACCESS-NRI/spack-packages
+      spack-config-version:
+        type: string
+        required: true
+        default: main
+        description: A version of ACCESS-NRI/spack-config
+jobs:
+  create-spack:
+    name: Spack
+    runs-on: ubuntu-latest
+    environment: "Gadi Spack"
+    steps:
+      - name: Setup SSH
+        id: ssh
+        uses: access-nri/actions/.github/actions/setup-ssh@main
+        with:
+          hosts: ${{ secrets.SSH_HOST }}
+          private-key: ${{ secrets.SSH_KEY }}
+      - name: Install
+        env:
+          SPACK_VERSION_LOCATION: ${{ vars.ROOT_SPACK_LOCATION }}/${{ inputs.spack-version }}
+          SPACK_LOCATION: ${{ env.SPACK_VERSION_LOCATION }}/spack
+          SPACK_PACKAGES_LOCATION: ${{ env.SPACK_VERSION_LOCATION }}/spack-packages
+          SPACK_CONFIG_LOCATION: ${{ env.SPACK_VERSION_LOCATION }}/spack-config
+          SPACK_RELEASE_LOCATION: ${{ env.SPACK_VERSION_LOCATION }}/release
+        run: |
+          ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
+          mkdir ${{ env.SPACK_VERSION_LOCATION }}
```

Comment:
I believe this will error if the directory already exists, and therefore the job will fail and not attempt subsequent steps. Can you confirm that is the case? I would like it to not work if the directory already exists.

Reply:
Good point. I believe it will attempt later steps, but will confirm. 
So just to confirm, if the directory exists (AKA, we have already deployed this version of spack), the job should fail.

Reply:
I believe so, because once it is deployed other deployments will use that location and build software (models and dependencies) in there. If we overwrite it it would be ... bad.

Maybe we would like an option to replace a deployment if something bad happens? That would be a (tactical) nuclear option ...

Reply:
I think if something that bad happened, we would be allowed to use the account to rectify it.

Reply:
Great, thanks.